crates/bevy_core_pipeline/src/bloom/bloom.wgsl

```diff
@@ -127,7 +127,7 @@ fn downsample(@location(0) uv: vec2<f32>) -> @location(0) vec4<f32> {
 
 @fragment
 fn upsample(@location(0) uv: vec2<f32>) -> @location(0) vec4<f32> {
```

Comment:
At this point, can we remove upsample_final? Seems like upsample and upsample_final both do the same thing now.

Reply:
That would be ideal. I am trying to come up with a mathematical representation of our interpolations that would first, remove any need for the final upsample pass, allow more artistic control, and double bloom as a free screen blur effect (can be combined with a texture to create blurred backgrounds for game UI)

I decided to keep changes to a minimum for now that's why it is there in this PR